user-interface/frontend/themes/datamanager/components/div.css

```diff
@@ -1,3 +1,42 @@
+.batch-details-component {
+  background-color: var(--lumo-base-color);
+  border: 1px solid;
+  border-color: var(--lumo-contrast-10pct);
+  border-radius: var(--lumo-border-radius-m);
+  margin: var(--lumo-space-s);
+  padding: var(--lumo-space-m);
+  flex-direction: column;
+  display: flex;
+  gap: var(--lumo-space-m);
+  /*ToDo How to best adapt this?*/
+  min-width: 500px;
+  min-height: 250px;
+  height: 25vh;
+  width: 40vw;
```

Comment:
Sadly Vaadin Grid needs absolute values from somewhere to initialize correctly. 
Otherwise it will default to 400px which is not scaleable to screen size. 
Following the flowchart the only scaleable solution seems to be VW and VH. 🤔